app/src/examples/RuntimeTests/tests/getRelativeCoords/relativeCoords.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { StyleSheet, ViewStyle } from 'react-native';
+import Animated, {
+  runOnUI,
+  measure,
+  getRelativeCoords,
+  ComponentCoords,
+  useAnimatedRef,
+  useSharedValue,
+} from 'react-native-reanimated';
+import {
+  describe,
+  test,
+  expect,
+  expectNotNullable,
+  render,
+  wait,
+  registerValue,
+  getRegisteredValue,
+} from '../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+const REGISTERED_VALUE_KEY = 'sv';
+
+const CoordsComponent = ({ justifyContent, alignItems }: { justifyContent: string; alignItems: string }) => {
```

Comment:
Instead of casting it below, you can restrict type here

```suggestion

const CoordsComponent = ({ justifyContent, alignItems }: { justifyContent: FlexStyle['justifyContent']; alignItems: FlexStyle['alignItems'] }) => {
```

 ViewStyle extends FlexStyle and I would recommend using FlexStyle here for simplicity, but it doesn't matter.